kadi/commands/__init__.py

```diff
@@ -1,2 +1,61 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+import logging
+
+logger = logging.getLogger(__name__)
+
+from astropy.config import ConfigNamespace
+from kadi.config import ConfigItem
+
+
+class Conf(ConfigNamespace):
+    """
+    Configuration parameters for my subpackage.
+    """
+    default_lookback = ConfigItem(
+        30,
+        'Default lookback for previous approved loads (days).'
+    )
+    cache_loads_in_astropy_cache = ConfigItem(
+        False,
+        'Cache backstop downloads in the astropy cache. Should typically be False, '
+        'but useful during development to avoid re-downloading backstops.'
+    )
+    clean_loads_dir = ConfigItem(
+        True,
+        'Clean backstop loads (like APR1421B.pkl.gz) in the loads directory that are '
+        'older than the default lookback. Most users will want this to be True, but '
+        'for development or if you always want a copy of the loads set to False.'
+    )
+    commands_dir = ConfigItem(
+        '~/.kadi',
+        'Directory where command loads and command events are stored after '
+        'downloading from Google Sheets and OCCweb.'
+    )
+    commands_version = ConfigItem(
+        '1',
+        'Default version of kadi commands ("1" or "2").  Overridden by '
+        'KADI_COMMANDS_VERSION environment variable.'
+    )
+
+    cmd_events_flight_id = ConfigItem(
+        '19d6XqBhWoFjC-z1lS1nM6wLE_zjr4GYB1lOvrEGCbKQ',
+        'Google Sheet ID for command events (flight scenario).'
+    )
+
+    star_id_match_halfwidth = ConfigItem(
+        5,
```

Comment:
For other work we've had ID_DIST_LIM at 1.5 I think.

Reply:
I was a little bit worried (without any particular evidence) about getting failed matches going way back in time e.g. to 2002. I guess for the most part all those catalogs went through starcheck and got independent cross-matching at 1.5 arcsec from starcheck. I'm fine dropping this to 1.5 arcsec.

Reply:
As follow-up I did comprehensive functional testing for mission and 1.5 arcsec is fine.